example.env

```diff
@@ -7,3 +7,4 @@ SERVER_URL=https://localhost/
 SLACK_WEBHOOK_URL=https://hooks.slack.com/services/T123456/B33Z/Pajamas
 SLACK_CHANNEL=channel
 LOG_FILE=/var/log/my-bot/chat.log
+NODE_PORT=-1
```

Comment:
In #50, I added a `test.env` that the test suite uses now. So this change should move there. Which is good because this file is supposed to be documentation for what the user is **supposed** to do. So I think this line can go into `test.env` and this file should change to:
```
# set NODE_PORT=-1 to disable the listener
NODE_PORT=3000
```

because this file now supports adding comments! 🎉